### PR TITLE
fix: Reword database cli prompts for clarity

### PR DESCRIPTION
**Pull request by deloristhompson**
Reword database cli prompts

## Changes

- Adds a prefix to each prompt

## Screenshots
Screen Shot 2020-12-31 at 10 43 14 AM

## Checklist
n/a

Fixes [Issue47](https://github.com/echobind/bisonapp/issues/47)

### CODE REVIEW

File: packages/create-bison-app/cli.js [hunk 1]
```diff
@@ -54,35 +55,40 @@ function generateQuestions(appName) {
     {
       name: "db.dev.name",
       type: "input",
-      message: "What is the local database name?",
+      prefix: "The following link <link> will help you setup a new Postgres database if needed.\n\bThe name you will use for the new database (or current).\n",
```
Comment (cball) [inline packages/create-bison-app/cli.js:58]: What do you think about asking if they have Postgres setup first?

> Do you have Postgres setup? y/n

(If the user answers no)
> Visit https://github.com/echobind/bisonapp/blob/main/docs/postgres.md for instructions!

(If the user answers yes, prompt for the database info below).

I like that you added both Homebrew and Postgres.app docs. Could we create a parent `Postgres` doc that links to each of those? That way, we can point users to a single place and they can choose how to proceed.

File: docs/homebrewPostgresInstructions.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Homebrew - (Mac Only)
+## Installing PostgreSQL
+
+1. Install using hombrew: `brew install postgres`
+2. Add a local user (we recommend creating as a superuser to keep setup simple):  `create user postgres --superuser`
```
Comment (mcavaliere) [inline docs/homebrewPostgresInstructions.md:5]: Doesn't the user called `postgres` exist by default? Either way I recommend calling it something different. 

It's worth also recommending use of Postgres.app, which I find way easier to work with. 


https://postgresapp.com/

File: docs/postgresAppInstructions.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Postgres.app (Mac Only)
```
Comment (mcavaliere) [inline docs/postgresAppInstructions.md:1]: These are great!

File: docs/postgresqlEDBInstructions.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+[PostgreSQL Database Download](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)
+
+## What is EDB Postgres?
+
+> A scalable platform designed to help users with everything from maintenance and distribution to support and documentation - *EDB.com*
+
+
+## Operating Systems:
+
+macOS
```
Comment (mcavaliere) [inline docs/postgresqlEDBInstructions.md:11]: Should this be a bulleted list maybe?

File: packages/create-bison-app/cli.js [hunk 5]
```diff
@@ -51,38 +52,58 @@ function generateQuestions(appName) {
...
+      message: `${chalk.green("\b?")} Local database host`,
       description: "The database host",
       default: "localhost",
     },
     {
       name: "db.dev.port",
       type: "input",
-      message: "What is the local database port?",
+      prefix: "The port you will use for the new database (or current).\n",
+      message: `${chalk.green("\b?")} Local database port`,
```
Comment (mcavaliere) [inline packages/create-bison-app/cli.js:106]: 👍 looking good

**Comment by cball** [comment]
We're going to be revisiting the CLI prompts / questions.

**Pull request closed without merging**